vllm/worker/enc_dec_model_runner.py

```diff
@@ -119,7 +118,11 @@ def __init__(
         # Crash for unsupported encoder/scenarios
         assert_enc_dec_mr_supported_scenario(self)
 
-    def _maybe_force_supported_attention_backend(self):
+    def _is_xformers_only_encoder_decoder_model(self, model: str) -> bool:
+        # The Llama 3.2 model implementation uses
+        return "llama-3.2-11b-vision-instruct" in model.lower()
```

Comment:
Another choice is to support llama vision in this pr. It shouldn't be too difficult.

Reply:
I added a TODO to support  llama vision with Flash Attention. I will address that in a separate PR. Is that fine?

Reply:
OK for me if you want to support llama vision in a separate PR. I address this here only because refactoring this check to an elegant way and removing it soon is a little time wasting.